packages/atlas/lib/src/marker.dart

```diff
@@ -76,3 +79,34 @@ class Anchor {
   /// {@macro anchor}
   const Anchor({this.x = 0.5, this.y = 0.5});
 }
+
+/// Text labels for a [Marker] info window.
+class CallOut {
+  final String title;
+
+  final String subTitle;
+
+  final MarkerIcon icon;
+
+  /// A `void Function` which is called whenever a `Marker info` is tapped.
+  final VoidCallback onTap;
+
+  const CallOut({
+    this.title,
+    this.subTitle,
+    this.icon,
+    this.onTap,
+  });
+
+  @override
+  bool operator ==(Object other) {
```

Comment:
This class can probably extend Equatable

Reply:
Yes, but it looks like all other classes (e.g. Marker, Polygon, etc) are not using Equatable right now. I think for consistency, we should keep it this way. And if we want to, we can make it a task to update them all at the same time.